test/site_selection.jl

```diff
@@ -34,12 +34,13 @@ end
 
 @testset "Guided site selection without ADRIA ecological model" begin
     dom = ADRIA.load_domain(joinpath(@__DIR__, "..", "examples", "Example_domain"), 45)
-    criteria_df = ADRIA.sample_site_selection(dom, 5)  # get scenario dataframe
+    N = 2^3
+    criteria_df = ADRIA.sample_site_selection(dom, N)  # get scenario dataframe
 
     area_to_seed = 1.5 * 10^-6  # area of seeded corals in km^2
     ts = 5  # time step to perform site selection at
 
-    sum_cover = 0.1 .* ones(5, size(dom.site_data, 1))
+    sum_cover = 0.1 .* ones(N, size(dom.site_data, 1))
```

Comment:
Didn't we replace this line with `fill(0.1, N, n_locations(dom))` ?

Reply:
I think we might have missed this (I did rebase with Main, so if it was replace it shoiuld be in here), I'll change it.